Week-9/TodoList/src/App.js

```diff
@@ -0,0 +1,113 @@
+import "./styles.css";
+import React, { useState } from "react";
+
+export default function App() {
+  const [inputList, setInputList] = useState([{ taskName: "" }]);
```

Comment:
So we have two lists just to track completion, could it not have been a single flag in the same state?